lib/rules/require-aligned-object-values.js

```diff
@@ -12,10 +40,13 @@ module.exports.prototype = {
             'skipWithFunction': 'ignoreFunction',
             'skipWithLineBreak': 'ignoreLineBreak'
         };
+
```

Comment:
@SgtPooki As a man who merging like thouthands line of code each 1-2 weeks I can answer you: it's just stupidly annoying when someone adding empty lines and another one removing them. Personally in my team we have non-verbal rules: 1. Don't add empty lines in places in the code unless you specially fixing code style; 2. Use separate issues and commits for code-style fixes; 3. Changes should be focused on issues.

I'm totally agree with @mikesherov that there are no reasons to toggle these empty lines in this issue.